src/python/pants/engine/scheduler.py

```diff
@@ -156,7 +157,7 @@ def _root_type_ids(self):
 
   def graph_trace(self, execution_request):
     with temporary_file_path() as path:
-      self._native.lib.graph_trace(self._scheduler, execution_request, bytes(path))
+      self._native.lib.graph_trace(self._scheduler, execution_request, bytes(path, 'utf-8'))
```

Comment:
I'm assuming `utf-8` is the correct encoding. It works with the tests, and I don't see any reason for us to use something else like `ascii`, but do want to flag this file.

Reply:
Yea, it should be. On osx and linux, paths are utf8.